src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/InnerAssignmentCheckTest.java

```diff
@@ -36,7 +36,7 @@ protected String getPackageLocation() {
     }
 
     @Test
-    public void testIt() throws Exception {
+    public void testInputInnerAssignment() throws Exception {
```

Comment:
The file name [InputInnerAssignment.java](https://github.com/checkstyle/checkstyle/blob/master/src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/innerassignment/InputInnerAssignment.java) doesn't really specify what the input file is actually testing.
You are required to analyse and rename such files according to test functionality as part of this issue....

I would recommend something like : `InputInnerAssignmentCheckLocalVariables`.java